docs/contents/components/forms/select/index.mdx

```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
 </Select>
 ```
 
+### Header
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  header={<header>Header added here</header>}
+>
+  <Option value="孫悟空">孫悟空</Option>
+  <Option value="ベジータ">ベジータ</Option>
+  <Option value="フリーザ">フリーザ</Option>
+</Select>
+```
+
+### Footer
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  footer={<footer>Footer added here</footer>}
```

Comment:
Change

```jsx
footer={<footer>Footer added here</footer>}
```

to

```jsx
footer={<Center pb="2" px="3">Footer added here</Center>}
```

Reply:
[f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)